crates/extensions/aws_mapper_ext/src/converter.rs

```diff
@@ -37,44 +37,51 @@ impl AwsConverter {
     }
 
     fn try_convert(&mut self, input: &MqttMessage) -> Result<Vec<MqttMessage>, ConversionError> {
-        let default_timestamp = self.add_timestamp.then(|| self.clock.now());
```

Comment:
[Minor] It could be nice to minimize the diff.

Here, one way to do that is to return early when this is a bridge topic.

```
// Ignore health status message coming from the bridges
if input.topic.name.starts_with("tedge/health") && input.topic.name.ends_with("bridge") {
    return vec![];
}

// ... former code unchanged
```